### PR TITLE
Tweak/pd 646 updates for overview

### PR DESCRIPTION
**Pull request by catherine-jones**

### CODE REVIEW

**Comment by elisechant** [comment]
`.updated-at` needs to be positioned to the bottom

**Comment by catherine-jones** [comment]
hey @elisechant happy to do that. Could you just let me know why so that I can improve in the future. Thanks :)

**Comment by elisechant** [comment]
@catherine-jones just thought that was the design. Noticed you don't have that part in your codes. Maybe it's not the design, im not completely sure? think i remember thats what it used to look like.

**Comment by catherine-jones** [comment]
service_dashboard_overview

@elisechant @phil-banks wanted to swap them around so that they were the same for mobile and desktop. Are you happy for this to be merged?

**Comment by elisechant** [comment]
either way, the bottom one is anchored to the bottom left.

**Comment by catherine-jones** [comment]
@elisechant ok I understand what you are saying let me double check with @phil-banks because he has already reviewed and was ok with this.

**Comment by phil-banks** [comment]
Hey @catherine-jones, sorry missed on review but the date has pushed the height of the card.  If I've got this correct, think that the date should live within the cards left panel and not on a row on its own, retaining the original height. @elisechant is that what you are referring too?

**Comment by elisechant** [comment]
@phil-banks yes it is. Just like in the screen grab.

**Comment by phil-banks** [comment]
ok cool @catherine-jones does this make sense to you?

**Comment by catherine-jones** [comment]
@elisechant @phil-banks yep think I understand thanks

File: client/src/dashboard/styles/dashboard/_card.scss [hunk 1]
```diff
@@ -65,18 +70,28 @@
...
 .card__right {
+
+  @media screen and (max-width: $screen-sm) {
+    padding-top: 20px;
+    border-top: 1px $axis-stroke-colour solid;
+    margin-top: 20px;
+  }
+
+  @media (max-width: 768px) {
+    .aus-width-one-fourth:last-child {
```
Comment (elisechant) [inline client/src/dashboard/styles/dashboard/_card.scss]: this is now bootstrap grid

**Comment by elisechant** [comment]
LGTM, grab @phil-banks to verify

File: client/src/dashboard/styles/dashboard/_card.scss (hunk 1 above)
Comment (catherine-jones) [inline client/src/dashboard/styles/dashboard/_card.scss]: moving this to bootstrap soon

**Pull request merged**